lib/tracing-metrics/src/lib.rs

```diff
@@ -1,222 +0,0 @@
-//! A `tracing` metrics based subscriber
```

Comment:
RIP old friend :pray:

Reply:
It will rise again from the ashes :fire: :bird: